blog/2022-06-30-paralus-quickstart-aks/index.md

```diff
@@ -0,0 +1,198 @@
+dashboard                       ClusterIP      10.0.134.169   <none>         80/TCP                          3m13s
+elasticsearch-master            ClusterIP      10.0.125.222   <none>         9200/TCP,9300/TCP               3m13s
+elasticsearch-master-headless   ClusterIP      None           <none>         9200/TCP,9300/TCP               3m13s
+myrelease-contour               ClusterIP      10.0.151.61    <none>         8001/TCP                        3m13s
+myrelease-contour-envoy         LoadBalancer   10.0.33.6      13.71.51.105   80:30193/TCP,443:30873/TCP      3m13s
+myrelease-kratos-admin          ClusterIP      10.0.16.90     <none>         80/TCP                          3m13s
+myrelease-kratos-courier        ClusterIP      None           <none>         80/TCP                          3m13s
+myrelease-kratos-public         ClusterIP      10.0.234.235   <none>         80/TCP                          3m13s
+myrelease-postgresql            ClusterIP      10.0.114.76    <none>         5432/TCP                        3m13s
+myrelease-postgresql-hl         ClusterIP      None           <none>         5432/TCP                        3m13s
+paralus                         ClusterIP      10.0.12.103    <none>         11000/TCP,10000/TCP,10001/TCP   3m13s
+prompt                          ClusterIP      10.0.50.46     <none>         7009/TCP                        3m13s
+relay-server      
+```
+
+Note down the IP address for the service type `LoadBalancer`
+
+Navigate to your domain's DNS setting page. _The steps for changing DNS settings will vary based on your domain name provider._
+
+While you are on your DNS Setting page, for the selected domain name, you need to add three A records. These will be based on the subdomains provided in the notes section post installation.
+
+| Type | Address | Resolves To | TTL |
+|---|---|---|---|
+| A | console-aks-oss.chartexample.com | 13.71.51.105 | 1 Hour |
+| A | *.core-connector.aks-oss.chartexample.com | 13.71.51.105 | 1 Hour |
+| A | *.user.aks-oss.chartexample.com | 13.71.51.105 | 1 Hour |
+
+### Accessing The Dashboard
+
+Paralus is installed with a default organization and an admin user. Hence, after installation, you need to set a password for the user. To do so, execute the command that you get after installing Paralus.
+
+```bash
+kubectl logs -f --namespace paralus $(kubectl get pods --namespace paralus -l app.kubernetes.io/name='paralus' -o jsonpath='{ .items[0].metadata.name }') initialize | grep 'Org Admin signup URL:'
+
+Org Admin signup URL:  http://console.chartexample.com/self-service/recovery?flow=de34efa4-934e-4916-8d3f-a1c6ce65ba39&token=IYJFI5vbORhGnz81gCjK7kucDVoiuQ7j
+
+```
+
+Access the URL in a browser, and provide a new password. In a new browser window/tab navigate to `http://console.chartexample.com` and log in with the following credentials:
+
+- username: `foo@example.com`
```

Comment:
It will be the the value they specific in the `values.yaml` file and by default it is `admin@paralus.local`.

Reply:
Got it.